flashinfer/decode.py

```diff
@@ -1817,6 +1819,8 @@ def _paged_run(
             max_seq_len,
             bmm1_scale,
             bmm2_scale,
+            -1,
```

Comment:
```suggestion
            -1, # o_sf_scale
            -1, # o_sf_vec_size
```
Add some comments when arguments meaning is not clear.